projects/admin/src/app/document-editor/document-editor.component.html

```diff
@@ -0,0 +1,46 @@
+<!--
+  RERO ILS UI
+   Copyright (C) 2019 RERO
+  
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3 of the License.
+  
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+  
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<header class="d-flex mb-4">
+  <h3 class="mb-0 mr-4">{{ recordType|translate }}</h3>
+  <button (click)="cancel()" class="btn btn-warning btn-sm" translate>Cancel</button>
+</header>
+
+<div class="form-group">
+  <label class="control-label" translate>Import record from BNF</label>
```

Comment:
You can retrieve translations from the `rero-ils/ui`.

Reply:
As I understand, the workflow for translation change as we use Transifex. So I just extract messages. Is that a way to add them in Transifex and translate the messages on it?

Reply:
In order to avoid the redo the translations you can copy and paste the translations from one project to the other.